CCache/Makefile.in

```diff
@@ -8,6 +8,7 @@ bindir=@bindir@
 mandir=@mandir@
 INSTALLCMD=@INSTALL@
 PACKAGE_NAME=@PACKAGE_NAME@
+MYNAME=@MYNAME@
```

Comment:
I also don't like `MYNAME`,  but the C sources use it (and it dosen't use `PACKAGE_NAME` except to define `MYNAME`, this is because I used `MYNAME` symbol directry). Any way,  I agree not to use `MYNAME` symbol except the C sources. I'm not sure what  changing value of `PACKAGE_NAME` affects, so I've tried to keep the value.

I looked at original ccache.h (at v2.4, v3.0pre0, v3.0, and master), apart from `#ifndef` block, they use embeded `"ccache"` C string to define `MYNAME`. So change to use `PROGRAM_NAME` isntead of `PACKAGE_NAME` here does'nt cause more divergence, I think.